tinygrad/jit.py

```diff
@@ -8,10 +8,10 @@
 
 class TinyJit:
   def __init__(self, fxn:Callable):
-    self.fxn = fxn
-    self.cnt = 0
+    self.fxn: Callable = fxn
+    self.cnt: int = 0
     self.jit_cache : List[Tuple[Callable, Any]] = []  # TODO: Any should be List[RawBuffer], but this fails
-    self.ret = None
+    self.ret: Optional[Tensor] = None
```

Comment:
This doesn't have to be a Tensor (fxn can return anything), Any is fine

Reply:
Also, match the "var : type" convention, not "var: type"

I'm open to changing this across the codebase if the other is more common